addon/utils/constants.js

```diff
@@ -0,0 +1 @@
+export const INVISIBLE_SPACE = '\u200B';
```

Comment:
just wondering: would it make sense importing this from the editor instead of a local constant?

Reply:
Is there a case where this invisible space constant would change?

Reply:
We can of course also import it from the editor

Reply:
I guess that depends on whether in the editor we consider these constants to be public (and thus part of our external API!) or not